Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.11.2-slim
+WORKDIR /app
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+RUN pip install --no-cache-dir poetry==1.5.1 \
+    && poetry config virtualenvs.create false
```

Comment:
Could you please clarify what should I change, because, to be honest, I don't get your point. 
As i see, i have COPY. . at the end of the Dockerfile, so all dependencies are installing before, as I understand this.